### PR TITLE
feat(noise-suppression): persist noise suppression setting

### PR DESCRIPTION
**Pull request by andrei-gavrilescu**

### CODE REVIEW

File: react/features/noise-suppression/functions.ts [hunk 1]
```diff
@@ -22,15 +22,6 @@ export function isNoiseSuppressionEnabled(state: IReduxState): boolean {
  * @returns {boolean}
  */
 export function canEnableNoiseSuppression(state: IReduxState, dispatch: IStore['dispatch'], localAudio: any): boolean {
-    if (!localAudio) {
-        dispatch(showWarningNotification({
-            titleKey: 'notify.noiseSuppressionFailedTitle',
-            descriptionKey: 'notify.noiseSuppressionNoTrackDescription'
```
Comment (saghul) [inline react/features/noise-suppression/functions.ts:28]: Can you pl drop these translations if they are now unused?

File: react/features/noise-suppression/reducer.ts [hunk 2]
```diff
@@ -8,17 +9,21 @@ export interface INoiseSuppressionState {
...
 
 /**
  * Reduces the Redux actions of the feature features/noise-suppression.
  */
-ReducerRegistry.register<INoiseSuppressionState>('features/noise-suppression',
+ReducerRegistry.register<INoiseSuppressionState>(STORE_NAME,
 (state = DEFAULT_STATE, action): INoiseSuppressionState => {
     const { enabled } = action;
 
+    PersistenceRegistry.register(STORE_NAME);
```
Comment (saghul) [inline react/features/noise-suppression/reducer.ts:25]: This can go outside, at the top level.

Comment (andrei-gavrilescu) [reply react/features/noise-suppression/reducer.ts:25]: moved

File: react/features/noise-suppression/functions.ts (hunk 1 above)
Comment (saghul) [reply react/features/noise-suppression/functions.ts:28]: Nah.

File: react/features/noise-suppression/reducer.ts (hunk 2 above)
Comment (saghul) [reply react/features/noise-suppression/reducer.ts:25]: can't see it, forgot to push?

Comment (andrei-gavrilescu) [reply react/features/noise-suppression/reducer.ts]: yea... sorry

**Review by saghul — APPROVED** [review]

**Pull request merged**